teacher_class.rb

```diff
@@ -0,0 +1,12 @@
+require './Person'
+
+class Teacher < Person
+  def initialize(age, name, specialization)
```

Comment:
- [ ] please make sure to include parent_permission in this initialize method as well otherwise you will get an error.